### PR TITLE
Use shared actions

### PR DESCRIPTION
**Pull request by adrienbernede**
Sharing Github Actions in a dedicated repo to avoid duplication with MFEM, Laghos and GLVis.

There should be no change in the CI results due to this PR.

Relates to:
https://github.com/mfem/mfem-actions
mfem/mfem#2146
GLVis/glvis#164
https://github.com/CEED/Laghos/pull/143

### CODE REVIEW

**Review by vladotomov — APPROVED** [review]

**Review by tzanio — APPROVED** [review]

**Review by artv3 — APPROVED** [review]

**Comment by adrienbernede** [comment]
I applied the suggested change of name for the action repo (https://github.com/mfem/mfem/pull/2146#issuecomment-816809576).
This uncovered a small bug, that was fixed in the action repo. Now waiting for tests to pass before I merge it.

**Pull request merged**
